lib/physical/shipment.rb

```diff
@@ -7,17 +7,23 @@ class Shipment
                 :destination,
                 :service_code,
                 :pallets,
+                :structures,
                 :packages,
                 :options
 
-    def initialize(id: nil, origin: nil, destination: nil, service_code: nil, pallets: [], packages: [], options: {})
+    def initialize(id: nil, origin: nil, destination: nil, service_code: nil, pallets: [], structures: [], packages: [], options: {})
       @id = id || SecureRandom.uuid
       @origin = origin
       @destination = destination
       @service_code = service_code
-      @pallets = pallets
+      @structures = Set[*structures]
       @packages = packages
```

Comment:
The commit message speaks about modifying the `#packages` method, but I don't see the modification.